releases/release-5.2.4.md

```diff
@@ -0,0 +1,197 @@
+---
+title: TiDB 5.2.4 Release Notes
+category: Releases
+---
+
+# TiDB 5.2.4 Release Notes
+
+发版日期：2022 年 4 月 xx 日
+
+TiDB 版本：5.2.4
+
+## 兼容性更改
+
++ TiDB
+
+    - (dup: release-5.1.4.md > 兼容性更改> TiDB)- 将系统变量 [`tidb_analyze_version`](/system-variables.md#tidb_analyze_version-从-v510-版本开始引入) 的默认值从 `2` 修改为 `1` [#31748](https://github.com/pingcap/tidb/issues/31748)
```

Comment:
```suggestion
    - 将系统变量 [`tidb_analyze_version`](/system-variables.md#tidb_analyze_version-从-v510-版本开始引入) 的默认值从 `2` 修改为 `1` [#31748](https://github.com/pingcap/tidb/issues/31748)
```

Reply:
We will remove the dup tag after the writer review is done to save some review effort.